app/src/main/java/org/listenbrainz/android/ui/screens/yim23/Yim23LandingScreen.kt

```diff
@@ -103,10 +104,10 @@ fun Yim23HomeScreen(
                                     Image(
                                         painter =
                                         painterResource(id = when (viewModel.themeType.value) {
-                                            0 -> R.drawable.yim23_pick_color_green
-                                            1 -> R.drawable.yim23_pick_color_red
-                                            2 -> R.drawable.yim23_pick_color_blue
-                                            3 -> R.drawable.yim23_pick_color_grey
+                                            Yim23ThemeData.greenTheme -> R.drawable.yim23_pick_color_green
```

Comment:
Rename the enum class and member themes as such:
```
enum class Yim23ThemeType(@DrawableRes val colorRes: Int) {
  GREEN(pickColorRes = R.drawable.yim23_pick_color_green, colorRes = ...),
  RED(...),
  BLUE(...),
  GREY(...)
}
```

Reply:
Ya thats so better.. will do that thx